content/docs/overview.md

```diff
@@ -1182,6 +1182,13 @@ Char_Set :: bit_set['A'..'Z'; u64];
 #assert(size_of(Char_Set) == size_of(u64))
 ```
 
+To get the number of elements in a bit set, use the `card` builtin procedure:
```

Comment:
To get the number of elements set, its cardinality, of a `bit_set`, use the built-in `card` procedure: